droid-hal-device-img-boot.inc

```diff
@@ -73,6 +75,12 @@ BuildRequires:  yamui
 BuildRequires:  openssh-clients
 BuildRequires:  openssh-server
 
+%if 0%{?embed_recovery:1}
+BuildRequires: shpp
+%else
+%define embed_recovery 0
```

Comment:
This renders some parts of the code unreachable, because defining something in .spec (even if equal to 0), counts as defined.
See here: https://backreference.org/2011/09/17/some-tips-on-rpm-conditional-macros/
and here: https://github.com/mer-hybris/droid-hal-device/blob/5fd1782c7f7b898fd66469d6cd43890c82219730/droid-hal-%40DEVICE%40.spec.template#L14